tests/bson/bson-utcdatetime-002.phpt

```diff
@@ -13,6 +13,6 @@ var_dump($utcdatetime);
 --EXPECTF--
 object(MongoDB\BSON\UTCDateTime)#%d (%d) {
   ["milliseconds"]=>
-  %rint\(|string\(13\) "|%r1416445411987%r"|\)%r
+  %rint\(|string\(10\) "|%r2147483647%r"|\)%r
```

Comment:
The `int|string` expectation dates back to https://github.com/mongodb/mongo-php-driver/commit/e497c0155cd16547f3f8a314c4a2ce8500b9484d. At the time, UTCDateTime's debug handler conditionally reported an int or string depending on the build architecture. That behavior was later changed in https://github.com/mongodb/mongo-php-driver/commit/d25e565f647d824ced02146bd41d8ef17ff93cb9 to always use strings.

Therefore, I think this can just expect:

```suggestion
  string(10) "2147483647"
```

Note: related to my earlier comment, I'm not sure if you decided to use `INT32_MAX` here instead of wrapping the original value with an Int64 object; however, since you'll probably change the expect pattern anyway I don't see the problem with changing the input value.

Reply:
Ah, yes. I've changed the expectf format to always expect a string.